src/tests/fica.test.ts

```diff
@@ -0,0 +1,113 @@
+import fc from 'fast-check'
+import { fica } from '../data/federal'
+import F1040 from '../irsForms/F1040'
+import F8959 from '../irsForms/F8959'
+import Form from '../irsForms/Form'
+import Schedule2 from '../irsForms/Schedule2'
+import Schedule3 from '../irsForms/Schedule3'
+import * as arbitraries from './arbitraries'
+
+function hasSSRefund (f1040: F1040): boolean {
+  const s3 = f1040.schedule3
+  const l10 = s3?.l10()
+  return l10 !== undefined && l10 > 0
+}
+
+function hasAdditionalMedicareTax (f1040: F1040): boolean {
+  const s2 = f1040.schedule2
+  const l8 = s2?.l8()
+  return l8 !== undefined && l8 > 0
+}
+
+type Constructor<T> = new (...args: any[]) => T
+function hasAttachment<FormType> (attachments: Form[], formType: Constructor<FormType>): boolean {
+  return attachments.find((f) => { return f instanceof formType }) !== undefined
+}
+
+describe('fica', () => {
+  it('should give refund SS tax overpayment only in some conditions', () => {
+    fc.assert(
+      fc.property(arbitraries.f1040, ([f1040, forms]) => {
+        if (f1040.w2s.length <= 1) {
```

Comment:
Notice if you change the <= here, or the threshold, or any <= type operator, you won't get this test to fail.

the `expect` function returns a matcher object, but doesn't run the actual test. Note in the jest source: 

```
    interface Expect {
        /**
         * The `expect` function is used every time you want to test a value.
         * You will rarely call `expect` by itself.
         *
         * @param actual The value to apply matchers against.
         */
        <T = any>(actual: T): JestMatchers<T>;
```

the expect interface gives you methods like `.toEqual(x: any): void` that will throw if the assertion is invalid, so functions in here instead of just `expect(!hasSSRefund(f1040))` need to be `expect(hasSSRefund(f1040)).toEqual(false)`

Reply:
good catch!

Reply:
Ah yes, thanks for the catch. I'll find time to update it later today.